core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -770,14 +761,21 @@ var ExplorationEditorMainTab = function() {
         parameterTypes[i])(parameterElement);
 
       if (interactionId === 'MultipleChoiceInput') {
-      // This is a special case as it uses a dropdown to set a NonnegativeInt.
-        await parameterElement.element(by.tagName('button')).click();
-        await multipleChoiceAnswerOptions(parameterValues[i])
-          .click();
+        // This is a special case as it uses a dropdown to set a NonnegativeInt.
+        var parameterElementButton = parameterElement.element(
+          by.tagName('button'));
+        await action.click('Parameter Element Button', parameterElementButton);
+        var multipleChoiceAnswerOption =
+          multipleChoiceAnswerOptions(parameterValues[i]);
+        await action.click(
+          'Multiple Choice Answer Option', multipleChoiceAnswerOption);
```

Comment:
Can you include the index of the answer option? That might help with debugging

Reply:
done